pnpm-workspace.yaml

```
@@ -0,0 +1,9 @@
+packages:
+  - .
+
+minimumReleaseAge: 10080
+strictDepBuilds: true
+allowBuilds: { esbuild: true, swiper: false }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)